src/main/resources/assets/skytils/lang/zh_TW.lang

```diff
@@ -327,7 +327,6 @@ skytils.config.slayer..current_sven_rng_meter=當前狼Slayer的RNG量表
 skytils.config.slayer..current_voidgloom_rng_meter=當前終界使者Slayer的RNG量表
 skytils.config.slayer..current_inferno_rng_meter=當前烈焰使者Slayer的RNG量表
 skytils.config.slayer..current_bloodfiend_rng_meter=當前吸血鬼Slayer的RNG量表
-skytils.config.slayer.quality_of_life.click_to_open_maddox_menu=點選任意處開啟Maddox選單
```

Comment:
Why was this line removed

Reply:
https://github.com/Skytils/SkytilsMod/pull/470

Maddox Menu was removed because Hypixel auto-opens the menu now

Reply:
Could you separate this into another pr? Would help with clarity.